IMG/cpio/ventoy/hook/debian/pve-wa.sh

```
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+script=$1
+tmpscript=${script}.tmp
+
+LINE=$(grep -n 'proc.*proc.*proc' $script | awk -F':' '{print $1}')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)